announcements/_posts/2020-03-11-call-for-imagine-2020-projects.md

```diff
@@ -1,9 +1,14 @@
 ---
 author: Justin W. Flory
-title: "Imagine RIT 2020: Call for projects"
+title: "CANCELED: Imagine RIT 2020: Call for projects"
 layout: post
 ---
 
+Unfortunately, [Imagine RIT 2020 was canceled](https://twitter.com/Imagine_RIT/status/1239558520984547328).
+We hope to see you next at [Maker Faire Rochester](https://rochester.makerfaire.com/) in November 2020!
+
+---
+
 Greetings!
 Are you working on an open source software or hardware project?
 Come present it with FOSS@MAGIC at Imagine RIT 2020!
```

Comment:
Do you still want to keep this part?

Reply:
Yeah. It's already published so I think it's fine to keep the existing content there with a disclaimer at the top.